addons/stock_dropshipping/stock_dropshipping.py

```diff
@@ -20,3 +21,15 @@ def _check_routing(self, product, warehouse):
                         res = True
                         break
         return res
+
+
+class purchase_order(models.Model):
+    _inherit = 'purchase.order'
+
+    @api.multi
+    def wkf_confirm_order(self):
+        for po in self:
+            if po.invoice_method == 'picking' and po.picking_type_id.name == 'Dropship' and\
```

Comment:
picking_type_id is not necessary and it is not a good practice to use name in this case (as somebody could for whatever reason call it 'immediate transfer to client' or something.  Best way to do this is to check if the procurement_ids of the purchase order lines have a sale_line_id and check the order policy there.  Searching through the name is not a good way to identify things.